Manipulators/StandardManipulator.cs

```diff
@@ -1,19 +1,29 @@
 ﻿#if UNITY_EDITOR
+using System;
 using System.Collections.Generic;
+using UnityEditor.Experimental.EditorVR.Core;
 using UnityEditor.Experimental.EditorVR.Handles;
 using UnityEditor.Experimental.EditorVR.Utilities;
 using UnityEngine;
 
 namespace UnityEditor.Experimental.EditorVR.Manipulators
 {
-	sealed class StandardManipulator : BaseManipulator
+	sealed class StandardManipulator : BaseManipulator, IControlHaptics, IRayToNode
```

Comment:
Since we already have the callbacks for translate and rotate, the subscriber of those events is what should be controlling haptics and not the manipulator itself. StandardManipulator is being treated as a dummy UI still.